NavBar and MainPage structure

mike2611 opened the pull request:
## NavBar
- Home/Main page.
- "Reserve" form.
- "My reservations".
- Add a room.
- Delete room.

## Main Page 
-  The user can see a list of the rooms

mike2611 committed:
Add router to navBar

mike2611 committed:
Install bootstrap for react

mike2611 committed:
Add boostrap-icon and create basic structure Home page

mike2611 committed:
Add mock fetching rooms

mike2611 committed:
Add styles to Main component

mike2611 committed:
Add button for pagination

mike2611 committed:
Add styles to navBar component

mike2611 committed:
Fix linter errors

mike2611 committed:
Fix merge conflict errors

LOctavio reviewed (commented):
# Good Job so far.

There is just one thing you should improve:
![localhost_3001_ (1)](https://user-images.githubusercontent.com/38335710/155206915-2de08a10-c10b-4906-9338-04a3dd36af0b.png)

The available Rooms view goes to the bottom when there are more than 3 rooms.

mike2611 committed:
Add pagination functionality to main page

mike2611 commented:
I added the pagination functionality  and fixed the bug
![image](https://user-images.githubusercontent.com/28109626/155239740-a49150e8-13bc-4f48-9c7a-b6cf266bcf04.png)

LOctavio reviewed (approved):
## Good job, you have successfully completed your task

### Status: Approved :green_circle:
